src/js/lightcase.js

```diff
@@ -549,7 +549,7 @@
 					break;
 				default:
 					if (_self.objectData.url) {
-						$object.load(function () {
+						$object.on('load', function() {
```

Comment:
@lbausch Thanks a lot for this pull request!
Just one thing: Anonymous functions are strictly witten with a space between `function ()` in lightcase.